browser/net/brave_torrent_redirect_network_delegate_helper.cc

```diff
@@ -44,34 +48,40 @@ bool IsMainFrameResource(std::shared_ptr<brave::BraveRequestInfo> ctx) {
 
 namespace webtorrent {
 
+bool ShouldRedirectRequest(
+    const net::HttpResponseHeaders* original_response_headers,
+    std::shared_ptr<brave::BraveRequestInfo> ctx) {
+  return !(!original_response_headers || !IsMainFrameResource(ctx) ||
```

Comment:
it's not clear why we should check `original_response_headers`

Reply:
TBH, I'm not sure either - I've just extracted this function from what was here before, and I'm hesitant to change it until I have a bit more time to sit down and test it.